components/director/internal/domain/scenarioassignment/service.go

```diff
@@ -150,60 +83,3 @@ func (s *service) List(ctx context.Context, pageSize int, cursor string) (*model
 
 	return s.repo.List(ctx, tnt, pageSize, cursor)
 }
-
-// DeleteManyForSameSelector missing godoc
-func (s *service) DeleteManyForSameTargetTenant(ctx context.Context, in []*model.AutomaticScenarioAssignment) error {
```

Comment:
The `Create`, `Delete`, `DeleteForTargetTenant` were moved from the ASA service to the formations service because it would lead to a circular dependency between the formations service, which uses those methods, the ASA engine, which is used in the ASA service and now, the ASA engine which is going to be use the formations service. The first two were renamed to `CreateAutomaticScenarioAssignment` and `DeleteAutomaticScenarioAssignment`. The repo layer is still used in the formations service.

Reply:
My comment was about different methods that the ones you mentioned. I didn't find where `DeleteForScenarioName`, `List` and `Create` are used?

Reply:
I think I misunderstood the initial comment, sorry. Those methods in the interface don't seem to be used here as it was practically split up. Those that are unused here are used in the formations service now, so these methods in the repo interface are unnecessary and will remove them, but the code in the repository layer is still needed as it's now used in the formations service. Only used methods seem to be `ListForTargetTenant`, `GetForScenarioName`, and `List`. Thanks, I was going to miss this.

Reply:
No worries ;) But now it seems the `ListAll` is not used in the `scenarioassignment/service.go file` ?

Reply:
After moving the rest of the methods for the `engine.go` it isn't used anymore. I just removed it from here, thanks.